src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -306,8 +318,22 @@
 
                 this.startPolling();
             },
-            toggleModal() {
-                this.open = !this.open;
+            openModal(){
+                if (this.selectedDataset && this.editsRequireConfirmation){
```

Comment:
I'm not convinced you need this check on selectedDataset - is editsRequireConfirmation not sufficient?